Tie account in system event subject to data and clean up subject names

matthiashanel opened the pull request:
Lookup is a request.
STATSZ was duplicated so it falls into this scheme: 
$SYS.REQ.SERVER.<ID>.<OPERATION> and $SYS.REQ.SERVER.<ID>.PING.<OPERATION>.
The original SYS.REQ.SERVER.<ID>.PING is kept for backwards compatibility. (reason for change in events_test.go)

As for update I didn't go all in on avoiding parsing the jwt twice as it' not a frequent operation and avoids code changes.

There are more changes to come in that area but these where the straight forward ones.
Signed-off-by: Matthias Hanel <mh@synadia.com>

kozlovic commented on server/accounts.go:
```diff
@@ -2937,7 +2937,8 @@ func (dr *DirAccResolver) Start(s *Server) error {
 	}
 	const accountPackRequest = "$SYS.ACCOUNT.CLAIMS.PACK"
-	const accountLookupRequest = "$SYS.ACCOUNT.*.CLAIMS.LOOKUP"
+	const accountLookupRequest = "$SYS.REQ.ACCOUNT.*.CLAIMS.LOOKUP"
```
Originally this was used only in this function, but now I wonder if we should move then to events.go (where you have defined number of tokens and index of account).
Also, we could make it a subject with the `%s` token, like `const accountLookupRequest = "$SYS.REQ.ACCOUNT.%s.CLAIMS.LOOKUP"`.
Here, to subscribe you would sprintf with this constant and pass "*" for the "%s".

matthiashanel replied:
will make these changes

kozlovic commented on server/accounts.go:
```diff
@@ -3070,7 +3071,7 @@ func (dr *CacheDirAccResolver) Fetch(name string) (string, error) {
 	}
 	respC := make(chan []byte, 1)
-	accountLookupRequest := fmt.Sprintf("$SYS.ACCOUNT.%s.CLAIMS.LOOKUP", name)
+	accountLookupRequest := fmt.Sprintf("$SYS.REQ.ACCOUNT.%s.CLAIMS.LOOKUP", name)
```
Here, I would use a different variable name and sprintf with the constant referenced earlier.

kozlovic reviewed (changes requested):
Would make this a constant subject defined in events.go and make use of it to create the subscription on "*" or lookup on specific account.

matthiashanel commented:
@kozlovic , incorporated your suggestions. also renamed claims.pack as it is a request as well. Moved the constant for it into system events just so we have one place where these can be found.

kozlovic reviewed (approved):
LGTM
